src/composables/useRightClickContextMenu.ts

```diff
@@ -0,0 +1,176 @@
+    const MAX_HEADER_LENGTH = 15;
+    const SHORTED_HEADER_FRAGMENT_LENGTH = 5;
+
+    willFocusOrBlur.value = true;
+
+    const getMenuItemButton = (label: string) => {
+      const item = contextMenudata.value.find((item) => item.label === label);
+      if (item?.type !== "button")
+        throw new Error("コンテキストメニューアイテムの取得に失敗しました。");
+      return item;
+    };
+
+    const text = inputSelection.getAsString();
+
+    if (text.length > MAX_HEADER_LENGTH) {
+      contextMenuHeader.value =
+        text.length <= MAX_HEADER_LENGTH
+          ? text
+          : `${text.substring(
+              0,
+              SHORTED_HEADER_FRAGMENT_LENGTH,
+            )} ... ${text.substring(
+              text.length - SHORTED_HEADER_FRAGMENT_LENGTH,
+            )}`;
+    } else {
+      contextMenuHeader.value = text;
+    }
+
+    if (inputSelection.isEmpty) {
+      getMenuItemButton("切り取り").disabled = true;
+      getMenuItemButton("コピー").disabled = true;
+    } else {
+      getMenuItemButton("切り取り").disabled = false;
+      getMenuItemButton("コピー").disabled = false;
+    }
+  };
+
+  const contextMenudata = ref<
+    [
+      MenuItemButton,
+      MenuItemButton,
+      MenuItemButton,
+      MenuItemSeparator,
+      MenuItemButton,
+    ]
+  >([
+    {
+      type: "button",
+      label: "切り取り",
+      onClick: async () => {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
+        contextMenu.value?.hide();
+        await handleCut();
+      },
+      disableWhenUiLocked: false,
+    },
+    {
+      type: "button",
+      label: "コピー",
+      onClick: async () => {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
+        contextMenu.value?.hide();
+        if (inputSelection) {
+          await navigator.clipboard.writeText(inputSelection.getAsString());
+        }
+      },
+      disableWhenUiLocked: false,
+    },
+    {
+      type: "button",
+      label: "貼り付け",
+      onClick: async () => {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
+        contextMenu.value?.hide();
+        await handlePaste();
+      },
+      disableWhenUiLocked: false,
+    },
+    { type: "separator" },
+    {
+      type: "button",
+      label: "全選択",
+      onClick: async () => {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
+        contextMenu.value?.hide();
+        qInputRef.value?.select();
+      },
+      disableWhenUiLocked: false,
+    },
+  ]);
+
+  const handleCut = async () => {
+    if (!inputSelection || inputSelection.isEmpty) return;
+
+    const text = inputSelection.getAsString();
+    const start = inputSelection.selectionStart;
+    setSurfaceOrYomiText(inputSelection.getReplacedStringTo(""));
+    await nextTick();
+    await navigator.clipboard.writeText(text);
```

Comment:
ここ元コードだとvoidっぽいですねぇ･･･。
意味あるのかわからないけど、とりあえず合わしときますか〜
```suggestion
    void navigator.clipboard.writeText(text);
```

Reply:
`void`に変更しました！